src/discord_voice_bot/voice/workers/synthesizer.py

```
@@ -19,13 +19,48 @@ class VoiceHandlerProtocol(Protocol):
     audio_queue: Any
     stats: Any
 
-    async def add_to_queue(self, message_data: dict[str, Any]) -> None: ...
+    async def add_to_queue(self, message_data: dict[str, Any]) -> None: """
+Enqueue a synthesis request for asynchronous processing.
```

Ruff (0.12.2)

Docstring is under-indented

Increase indentation

(D207)